libs/handoverapp/src/lib/config/statusBarConfig.ts

```diff
@@ -1,48 +1,20 @@
-import { HandoverPackage } from '@cc-components/handovershared';
-import { numberFormat } from '@cc-components/shared/utils-formatting';
+import { useHttpClient } from '@equinor/fusion-framework-react-app/http';
 import { StatusBarConfig } from '@equinor/workspace-fusion/status-bar';
-import { getStatusBarData } from '../utils-status-bar';
-export const statusBarConfig: StatusBarConfig<HandoverPackage> = (data) => {
-  const statusData = getStatusBarData(data);
 
-  return [
-    {
-      title: 'Total CP',
-      value: numberFormat(data.length),
-    },
-    {
-      title: 'RFO Accepted',
-      value: numberFormat(statusData['RFOC Accepted']),
-    },
-    {
-      title: 'RFO Sent',
-      value: numberFormat(statusData['RFOC Sent']),
-    },
-    {
-      title: 'RFO Partly',
-      value: statusData['RFOC Partly'].toString(),
-    },
-    {
-      title: 'RFO OS',
-      value: numberFormat(statusData.OS),
-    },
+export const useStatusBarConfig = (contextId: string): StatusBarConfig => {
+  const client = useHttpClient('cc-app');

```

Comment:
So to avoid possible typos and fatal runtime errors you can use useHttpClient from `import { useHttpClient } from '@cc-components/shared';`

The only place you need the name of the client is in frameworkConfig.ts and main.tsx where you pass it into the rootAppWrapper